### PR TITLE
Relnotes2017.25

### PR DESCRIPTION
**Pull request by richardmateosian-okta**
## Description:
Complete first draft for review
## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* https://oktainc.atlassian.net/browse/OKTA-130808)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
Doc

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
There's too many commits in this PR. I'll mark it up, but you need to rebase, I think.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 1]
```diff
@@ -1,111 +1,80 @@
...
-    be warned these rate limits will be enforced in the near future.
-    ```
+#### Create OAuth 2.0 and OpenID Connect clients with Apps API
+You can create and configure OAuth 2.0 or OpenID Connect clients with the Apps API. <links to each - TODO>
+Previously this was only available in the <beta tag - TODO> Client Registration API.
+<!-- (OKTA-78223) -->
 
-2. As of mid-May, we started enforcing these new rate limits for all newly created orgs. For these new orgs, instead of alerts in your System log, the API calls exceeding the new rate limits return an HTTP 429 error.
+#### HTTP 401 response includes scheme in WWW-Authenticate header
+When Okta returns an invalid_client error it returns the WWW-Authenticate header with the value of
```
Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: `invalid_client`  Wondering if we should simply call this the authentication header as we do in the rest of the doc? Is it the same header?

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 2]
```diff
@@ -1,111 +1,80 @@
...
+the supported authentication scheme in the HTTP 401 response. <link to spec? - TODO>.
+<!-- (OKTA-127653) -->
```
Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: link to spec not needed in RN, I think that was the gist of Cindy's comment.

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: My bad for suggesting them!

**Comment by richardmateosian-okta** [comment]
I screwed this up. I'm starting over.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 3]
```diff
@@ -1,111 +1,80 @@
...
 
-##### Preview Orgs
+#### Key Rollover Change
+The `credentials.signing.kid` property of an app is available only if the app supports the key rollover feature,
+that is, if it uses one of the following signing mode types: SAML 2.0, SAML 1.1, WS-Fed, or OpenID Connect.
+<!-- OKTA-76439 -->
 
-We enforce new rate limits for all preview orgs. API calls exceeding the new rate limits return an HTTP 429 error.
+#### System logs track key rotation and generation
+Logged information about key rotation and generation for `idp/app` instances is available by using GET requests
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: replaced "idp/app" with "identity provider and application"

The only place the word "logged" appears is at the beginning of the sentence. It removes the need to refer to system logs.

Shortened the heading

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 4]
```diff
@@ -1,111 +1,80 @@
...
+to either of the following endpoints: `/api/v1/events` or `/api/v1/logs`.
+<!-- (OKTA-76607) -->
 
-##### Production Orgs
+#### Client app updates available from events endpoint
+Logged information about OAuth 2.0 client updates is now available by using GET requests to
+the `/api/v1/events` endpoint. Previously such information was available only from the `/api/v1/logs` endpoint.
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 5]
```diff
@@ -1,111 +1,80 @@
...
+<!-- (OKTA-86738, OKTA-127445) -->
 
-1. As of May 8, we enabled a System Log alert which lets you know if you have exceeded any of the new API rate limits:
+#### Support for RP-initiated logout
+Okta supports [RP-intiated logout](http://openid.net/specs/openid-connect-session-1_0.html#RPLogout)
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I don't see such a comment in OKTA-94106

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 6]
```diff
@@ -1,111 +1,80 @@
...
+from OpenID Connect client apps in both the Okta UI and Okta API. You can specify a logout redirect URI,
+or accept the default behavior of returning to the Okta Login page. For more information,
+see <prod doc link - TODO> or <api doc link - TODO>.
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I think that's what I did when I added the actual links.

Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK. I've referred them to the Create OpenID Connect Integration page (under Applications) in the UI.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 7]
```diff
@@ -1,111 +1,80 @@
...
 
-    ```
-    Warning: requests for url pattern <url-pattern> have reached 
-    a threshold of <number> requests per <time-duration>. Please 
-    be warned these rate limits will be enforced in the near future.
-    ```
+#### Create OAuth 2.0 and OpenID Connect clients with Apps API
+You can create and configure OAuth 2.0 or OpenID Connect clients with the Apps API. <links to each - TODO>
+Previously this was only available in the <beta tag - TODO> Client Registration API.
+<!-- (OKTA-78223) -->
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK

File: _source/_docs/platform-release-notes/platform-release-notes.md (hunk 2 above)
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I took it out.

File: _source/_docs/platform-release-notes/platform-release-notes.md (hunk 1 above)
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: WWW-Authenticate is the name the spec uses for the header. I don't know how that relates to "the authentication header"

**Comment by mystiberry-okta** [comment]
Doesn't need to be addressed this time, but let's use short noun phrases in list intros instead of semantically empty phrases to create a full sentence. I'd like that to be our style going forward.

so, instead of "Here's an example of a system log entry," simply "System Log Example"

If the list *needs* an intro explanation, that's fine too.

**Pull request merged**